app/templates/Articles/add.php

```diff
@@ -0,0 +1,12 @@
+<!-- File: templates/Articles/add.php -->
+
+<h1>Add Article</h1>
+<?php
+    echo $this->Form->create($article);
+    // Hard code the user for now.
+    echo $this->Form->control('user_id', ['type' => 'hidden', 'value' => 1]);
+    echo $this->Form->control('title');
+    echo $this->Form->control('body', ['rows' => '3']);
+    echo $this->Form->button(__('Save Article'));
+    echo $this->Form->end();
+?>
```

Comment:
I thought a php code needed it, but after reading is not mandatory, also the cs-check or stand is not complaining about it

Reply:
> I thought a php code needed it, but after reading is not mandatory

I'm sorry I didn't understand the part of is not mandatory. Can you explain more? Thank you.

> also the cs-check or stand is not complaining about it

Our configuration for code sniffer didn't check the templates folder. We only apply the checking for folders `src` & `tests`. Refer https://github.com/ArielGalvez/tutorial-cakephp/blob/99ab9782629c231533ee7d746dc6cd697f964773/app/phpcs.xml#L7-L8

Reply:
![image](https://github.com/ArielGalvez/tutorial-cakephp/assets/32244920/ce83dcab-3444-4bcf-bd78-e06400af0bf5)
When I said 'mandatory' I was referring to the question mark as closing PHP code (the red square in the picture). I also changed to LF, as you can see (green square).

Reply:
Thank you for your answer. 

It's referring to closing `?>` tag. From https://www.php-fig.org/psr/psr-12/#22-files

> The closing ?> tag MUST be omitted from files containing only PHP.

The rules mention MUST omit when the file containing **ONLY** PHP. For this template, it's mixing the PHP code and HTML syntax. Therefore the closing `?>` tag is needed in this case.

For example:
1. NG (Missing closing tag):
    ```
    <h1>Title</h1>
    <?php
        echo 'ABC';
    ```
2. OK (Omit closing tag as the file is containing only PHP code):
    ```
    <?php
        echo 'ABC';
    ```

The original issue on ⛔ symbol displayed because it's missing the newline at the end of the file. I'm referring to

> All PHP files MUST end with a non-blank line, terminated with a single LF.

Please check.

Reply:
if you see in the screenshoot in the green square is already in LF